gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/documentation/domain_service/ApiDocumentationDomainService.java

```diff
@@ -59,13 +47,6 @@ public void validatePageAssociatedToApi(Page page, String apiId) {
         }
     }
 
-    public void validateContentIsSafe(String content) {
-        final SanitizeResult sanitizeInfos = htmlSanitizer.isSafe(content);
-        if (!sanitizeInfos.isSafe()) {
-            throw new PageContentUnsafeException(sanitizeInfos.getRejectedMessage());
-        }
-    }
-
     public void validateNameIsUnique(String apiId, String parentId, String name, Page.Type type) {
```

Comment:
Why not move this method as well to the `DocumentationValidationDomainService`?

Reply:
I move the validate content method because it's not linked to the Api.